stores/moderationSubmissionsStore.ts

```diff
@@ -17,6 +17,22 @@ export const useModerationSubmissionsStore = defineStore(
         const selectedSubmissionId: Ref<string> = ref('')
         const selectedSubmissionData: Ref<Submission | undefined> = ref()
         const filteredSubmissionDataForListComponent: Ref<Submission[]> = ref([])
+        const updateSubmissionFromTopNav: Ref<boolean> = ref(false)
+        const isSendingUpdatedForm: Ref<boolean> = ref(false)
+        const didMutationFail: Ref<boolean> = ref(false)
+        const mutationAttempts: Ref<number> = ref(0)
+
+        function handleUpdateSubmissionFromTopNav(newValue: boolean) {
+            updateSubmissionFromTopNav.value = newValue
+        }
+
+        function handleUpdateIsSendingUpdatedForm(newValue: boolean) {
+            isSendingUpdatedForm.value = newValue
+        }
+
+        function handleDidMutationFail(newValue: boolean) {
+            didMutationFail.value = newValue
```

Comment:
maybe these could be combine into a `formSubmissionStatus` with `sending`, `failed`, etc.

Reply:
@ermish I will make this a draft again since I am going to wait for the retry logic from the rejection usable function to be implemented